ASFAchievementManager/AchievementHandler.cs

```diff
@@ -20,23 +20,21 @@ public sealed class AchievementHandler : ClientMsgHandler {
 			switch (packetMsg.MsgType) {
 				case EMsg.ClientGetUserStatsResponse:
 					ClientMsgProtobuf<CMsgClientGetUserStatsResponse> getAchievementsResponse = new(packetMsg);
-					Client.PostCallback(new GetAchievementsCallback(packetMsg.TargetJobID, getAchievementsResponse.Body));
-
+					Client.PostCallback(new GetAchievementsCallback(packetMsg.TargetJobID, getAchievementsResponse.Body, (msg) => msg.eresult));
```

Comment:
You'll need to remove the lambdas here if you do what I said in the comment on the constructors